sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/schema.yaml

```diff
@@ -2303,3 +2306,30 @@ fields:
 - name: startup_profile_selection_reason_first
   type: STRING
   mode: NULLABLE
+- name: first_document_id
+  type: STRING
+  mode: NULLABLE
+- name: partner_id
+  type: STRING
+  mode: NULLABLE
+- name: distribution_version
```

Comment:
It would be lovely to have descriptions for these

Reply:
I agree as you know, but in the case of `clients_daily` which currently has.. zero descriptions I'm slightly more 🤷‍♂️